### DISCUSSION TITLE
Notifications of ICU4X releases.

### DISCUSSION THREAD

**Discussion by rizzen-yazston** [unicode-org/icu4x#3239] [Q&A] [answered]
I have recently completed my library that uses ICU4X as its backend, and busy finalising the package in order to upload to crates.io for anyone to use.

Currently my library is using ICU4X 1.1.0, thus my library is still incomplete due to some ICU4X components are still in an experimental state. Though the library is currently using the experimental grapheme cluster segmenter, even though still having issues to count the graphemes in a string. I decided to include it as to have my token component to be complete structurally.

I would like to update my library as experiential components become stable in new releases of ICU4X.

What are the recommended options for obtaining notifications of new ICU4X releases?

**Comment by zbraniecki** [comment, marked as answer]
Follow the Github "Watch" feature - you can select to follow releases.

**Comment by sffc** [comment]
In addition to GitHub notifications, you can join the [icu4x-announce](https://groups.google.com/u/0/a/unicode.org/g/icu4x-announce) mailing list.
